### PR TITLE
Fixes #17280 - Update dep in scap spec file

### PR DESCRIPTION
**Pull request by chris1984**
For plugin updates, please indicate which repos this should be built into:

* [ ] Nightly
* [ ] 1.13
* [x] 1.12
* [ ] 1.11

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by chris1984** [comment]
@domcleal I closed the other PR and created this new one, I updated the issue showing that the build with the changed require in the spec file is giving the same output which is good.

File: rubygem-foreman_scap_client/rubygem-foreman_scap_client.spec [hunk 1]
```diff
@@ -16,7 +16,7 @@ Requires: ruby(rubygems)
 %if 0%{?rhel} >= 5 && 0%{?rhel} <= 6
 BuildRequires: ruby(abi) >= %{rubyabi}
 Requires: ruby(abi) >= %{rubyabi}
-Requires: openscap-utils
+Requires: openscap-scanner
```
Comment (domcleal) [inline rubygem-foreman_scap_client/rubygem-foreman_scap_client.spec]: Would /usr/bin/oscap be preferable here, for compatibility with older versions?

**Comment by chris1984** [comment]
@domcleal does it look ok now for a merge?

**Comment by chris1984** [comment]
I tested it with the change and still looks good when installing on a EL6 box

**Comment by domcleal** [comment]
Merged and will build, thanks @chris1984. Commit message fixed.

**Pull request merged**